src/common/meta/src/rpc/ddl.rs

```diff
@@ -645,7 +645,7 @@ pub struct CreateDatabaseTask {
     pub catalog: String,
     pub schema: String,
     pub create_if_not_exists: bool,
-    pub options: Option<HashMap<String, String>>,
+    pub options: HashMap<String, String>,
```

Comment:
@WenyXu Does it have a compatibility issue as structs like `CreateDatabaseTask` derive `Serialize` and `Deserialize`. We may encounter an error if we deserialize the map from `null`.

Reply:
> @WenyXu Does it have a compatibility issue as structs like `CreateDatabaseTask` derive `Serialize` and `Deserialize`. We may encounter an error if we deserialize the map from `null`.

Let's add the `#[serde_as(deserialize_as = “DefaultOnNull”)]`

Reply:
Make sense. Confirm that `null` can be a compatibility issue.

Reply:
Updated.